File: day36/classwork/cw.py
#შექმენით ფუნქცია რომელსაც ექნება ორი პარამეტრი ხოლო ამ ფუნქციამ უნდა გადაამრავლოს ეს ორი რიცხვი ერთმანეთზე შემდეგ კი დაიბეჭდოს მიღებული ნარმავლი ლუწია თუ კენტი ნამრავლთან ერთად, გამოიძახეთ ეს ფუნქცია რამდენჯერმე და გადაეცით არგუმენტები(სვადასვა რიცხვები)

def func(num1, num2):
    num1 = int(input("dawere"))
    num2 = int(input("dawere"))
    print(num1 * num2)
    if num1 or num2 % 2 ==0:
        print("luwi")
    else:
        print("kenti")


func("kentia" , "luwia")

File: day110/classwork/cw.js
//შექმენით სია რომელშიც იქნება რიცხვები, შემდეგ შექმენით ორი დამატებითი სია, ერთ სიაში ჩაამატეთ მთავარი სიიდან მხოლოდ ლუწი რიცხვები ხოლო მეორეში მხოლოდ კენტები.



let number = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];

let luwi = [];
let kenti =[];


for(let i =0;  i < number.length; i+=1) {
    if(number[i] % 2 === 0) {
        luwi.push(number[i]);
    } else{
        kenti.push(number[i]);
    }
}

console.log(luwi);
console.log(kenti);